packages/app-studio-toolkit/test/project-type/workspace-instance.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { mockVscode } from "../mockUtil";
+import { ok } from "assert";
```

Comment:
can you please use chai for assertions
to stay consistent with the rest of this project?
`import { expect } from "chai"`

I think it is also a much richer assertion library
- https://www.chaijs.com/api/assert/

Reply:
It has a `match` utility on strings
- expect('foobar').to.match(/^foo/);

Reply:
Yes, it is better choice to use chai here. I will replace assert with chai in this file.